### PR TITLE
Rbd_utils_refactor Version1

### PR DESCRIPTION
**Pull request by HaruChebrolu**
Signed-off-by: Chebrolu Harika <hchebrol@redhat.com>

# Description

Rbd_utils_refactor Version1

lib folder contains all SDK calls for RBD
tests/rbd/test_configs contains testcase specific workflows

Scenario covered: Snap and clone operations on imported image
Conf file used: conf/pacific/rbd/tier-0_rbd.yaml
Command used: python run.py --osp-cred osp/osp-cred-ci-2.yaml --log-level debug --build latest --inventory conf/inventory/rhel-8-latest.yaml --global-conf conf/pacific/rbd/tier-0_rbd.yaml --suite suites/pacific/rbd/tier-2_rbd_mirroring.yaml --rhbuild 5.1 --platform rhel-8

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: conf_testcases/test_rbd_benchwrite.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  - method: shell
+    command: 'sleep 15'
+  - method: wait_for_operation
+    class: rbdmirror
+    args:
+      response: "up+stopped"
+  - method: benchwrite
+    class: rbdmirror
+    args:
+      total-io: '1G'
```
Comment (psathyan) [inline conf_testcases/test_rbd_benchwrite.yaml:28]: As a good practice end with a newline character.

File: lib/rbd/rbd_mirroring.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+from helpers.rbd import Rbd
```
Comment (psathyan) [inline lib/rbd/rbd_mirroring.py:1]: not sure were `helpers.rbd`

File: conf_testcases/test_rbd_benchwrite.yaml (hunk 1 above)
Comment (HaruChebrolu) [reply conf_testcases/test_rbd_benchwrite.yaml]: done

File: lib/rbd/rbd_mirroring.py (hunk 2 above)
Comment (HaruChebrolu) [reply lib/rbd/rbd_mirroring.py]: changed it.

File: conf_testcases/test_rbd_benchwrite.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+name: test-image-shrink
```
[the remark this answers is not in the document]
Comment (HaruChebrolu) [reply conf_testcases/test_rbd_benchwrite.yaml]: done

**Pull request closed without merging**